From 0c9305bec7d1061538368179e3d62d643900eaed Mon Sep 17 00:00:00 2001
From: Priyanka Jain <priyanka.jain@nxp.com>
Date: Tue, 8 May 2018 11:13:14 +0530
Subject: [PATCH 054/107] boards: freescale: lsch3: Add 0.9v entry for vid

As per updated hardware documentation for
lsch3 based chips like LS2088A, 0.9v support
has been added in possible supported SoC volatge
in vid table

Signed-off-by: Priyanka Jain <priyanka.jain@nxp.com>
---
 board/freescale/common/vid.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/board/freescale/common/vid.c b/board/freescale/common/vid.c
index a9451c5..06edc32 100644
--- a/board/freescale/common/vid.c
+++ b/board/freescale/common/vid.c
@@ -428,7 +428,7 @@ int adjust_vdd(ulong vdd_override)
 		0,      /* reserved */
 		0,      /* reserved */
 		0,      /* reserved */
-		0,      /* reserved */
+		9000,      /* reserved */
 		0,      /* reserved */
 		0,      /* reserved */
 		0,      /* reserved */
-- 
1.7.1

